remoteit/desktop
# [DESK-618] Display session info

### Description


Now that we can GraphQL data in Desktop it's easy to show the user that have access as well as the connected users.

### Checklist




### Test plan



1. Sign in with account A
2. Connect to a device from account B
3. On another machine with the desktop app sign in with accound B
4. You should see account A connected

### Ticket



<https://remoteit.atlassian.net/browse/DESK-618>
